vrecord

```diff
@@ -486,6 +486,14 @@ yes_or_no(){
     esac
 }
 
+_frames_to_hhmmss(){
+    num=`expr $i / $framerate`
+    h=`expr "$num" / 3600`
+    m=`expr "$num"  % 3600 / 60`
+    s=`expr "$num" % 60`
```

Comment:
I suggest to use the `$( . . . )` syntax here as well, like at line 1317 and following.